h2o-algos/src/main/java/hex/schemas/StackedEnsembleV99.java

```diff
@@ -171,6 +173,32 @@ public StackedEnsembleParametersV99 fillFromImpl(StackedEnsembleModel.StackedEns
             paramsSchema = new DeepLearningV3.DeepLearningParametersV3();
             params = new DeepLearningModel.DeepLearningParameters();
             break;
+          case naivebayes:
+            paramsSchema = new NaiveBayesV3.NaiveBayesParametersV3();
+            params = new NaiveBayesModel.NaiveBayesParameters();
+            break;
+          case xgboost:
+            try {
+              ClassLoader classLoader = getClass().getClassLoader();
+              paramsSchema = (ModelParametersSchemaV3)  classLoader
+                      .loadClass("hex.schemas.XGBoostV3$XGBoostParametersV3")
+                      .getConstructor()
+                      .newInstance();
+              params = (Model.Parameters) classLoader
+                      .loadClass("hex.tree.xgboost.XGBoostModel$XGBoostParameters")
+                      .getConstructor()
+                      .newInstance();
+            } catch (ClassNotFoundException |
+                     NoSuchMethodException |
+                     IllegalAccessException |
+                     InstantiationException |
+                     InvocationTargetException e) {
+              e.printStackTrace();
+            }
```

Comment:
oh I see, you tried with reflection.
Yeah, the `Metalearners` approach is definitely more elegant.

btw, no `e.printStackTrace()`, use logger instead.

Reply:
Thanks!